src/components/comparison/layers-panel/layers-panel.tsx

```diff
@@ -235,13 +288,37 @@ export const LayersPanel = ({
               />
             </PanelWrapper>
           )}
-
-          {showInsertPanel && (
+          {showExistedSceneWarning && (
+            <PanelWrapper ref={(element) => setWarningNode(element)}>
+              <WarningPanel
+                title={EXISTING_AREA_WARNING}
```

Comment:
The conten of the warning is the same as for `showExistedLayerWarning`. We could use `showExistedLayerWarning` and its warning for the insert scene.